x-pack/plugins/actions/server/manual_tests/forward_proxy.js

```diff
@@ -34,11 +34,13 @@ echo done - you should run all the lines above as one command
 
 */
 
+// @ts-check
```

Comment:
This enables VS Code to do type-checking on the file.  I don't think it will be type-checked during ci, but at least while someone is editing it they should get feedback on typing issues.